### DISCUSSION TITLE
请问Stanford Alpaca格式的指令精调如何支持多轮会话

### DISCUSSION THREAD

**Discussion by Terminaldienst** [ymcui/Chinese-LLaMA-Alpaca-2#104] [Q&A] [answered]
我看了下，Stanford Alpaca好像只支持一轮对话，有什么输入多轮对话进行精调的方法吗，感谢分享

**Comment by airaria** [comment, marked as answer]
按Llama-2的官方格式进行拼接：https://github.com/facebookresearch/llama/blob/main/llama/generation.py

**Reply by Terminaldienst** [reply]
感谢，我研究研究

**Comment by GJZ0819** [comment]
请问楼主解决这个问题了吗？
